test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultRazorIndentationFactsServiceTest.cs

```diff
@@ -167,6 +167,35 @@ public void GetIndentLevelOfLine_NoIndent()
             Assert.Equal(0, indentLevel);
         }
 
+        [Fact]
+        public void GetDesiredIndentation_ReturnsNull_IfOwningSpanDoesNotExist()
+        {
+            // Arrange
+            var tagHelper = TagHelperDescriptorBuilder.Create("TestTagHelper", "TestAssembly");
+            tagHelper.TagMatchingRule(builder => builder.RequireTagName("testtaghelper"));
+            tagHelper.SetTypeName("TestTagHelper");
+
+            var source = new StringTextSnapshot($@"
+<div>
+    <div>
+    </div>
+</div>
+");
+            var syntaxTree = GetSyntaxTree(new StringTextSnapshot("something else"));
+            var service = new DefaultRazorIndentationFactsService();
+
+            // Act
+            var indentation = service.GetDesiredIndentation(
+                syntaxTree,
+                source,
+                source.GetLineFromLineNumber(3),
+                indentSize: 4,
+                tabSize: 1);
```

Comment:
I don't super understand this repo - so you're creating a source document and syntax tree that don't match?

Reply:
Sorry I should have been more clear in this PRs comments. This is a quick overly protective fix that accomplishes the following:
1. Protects us against a WTE bug where they ask us for indentation for a line that doesn't exist in our last parsed snapshot.
2. We should be protecting against unknown owners anyways because we know we have gaps in our syntax tree.